pkg/manager/member/tikv_scaler_test.go

```diff
@@ -252,19 +252,6 @@ func TestTiKVScalerScaleIn(t *testing.T) {
 			errExpectFn:   errExpectRequeue,
 			changed:       false,
 		},
-		{
-			name:          "tikv is upgrading",
```

Comment:
Can we modify the case to cover the scenario that scaling can be done during upgrading?

Reply:
updated